packages/validator/src/commands/utils.ts

```diff
@@ -49,3 +66,68 @@ export function getPath(template: string, messages: Log[], version = ''): string
 
   return res;
 }
+
+/**
+ * Returns path to conversion template between specified formats
+ * @param from - name of source specification format
+ * @param to - name of target specification format
+ * @param messages - array into which warnings/errors will pushed
+ * @param fromVersion (optional) - version of source specification format, if not specified, latest version of the template is returned
+ * @param toVersion (optional) - version of target specification format, if not specified, latest version of the template is returned
+ */
+export function getConversionPath(from: string, to: string, messages: Log[], fromVersion = '', toVersion = ''): string {
+  if (!conversions[from]) {
+    messages.push({ level: 'error', message: `Unknown conversion from ${from} to ${to}` });
+    return '';
+  }
+
+  if (!fromVersion) {
+    let fromLatest;
+
+    for (const version in conversions[from]) {
+      if (!conversions[from][version][to]) {
+        continue;
+      }
+
+      fromLatest = !fromLatest || (fromLatest < version && version.match(/^[0-9\.]+$/)) ? version : fromLatest;
+    }
+
+    if (!fromLatest) {
+      messages.push({ level: 'error', message: `Unknown conversion from ${from} to ${to}` });
+      return '';
+    }
+
+    fromVersion = fromLatest;
+  }
+
+  if (!conversions[from][fromVersion][to]) {
+    messages.push({ level: 'error', message: `Unknown conversion from ${from} to ${to}` });
+    return '';
+  }
+
+  if (!toVersion) {
+    let toLatest;
+
+    for (const version in conversions[from][fromVersion][to]) {
+      toLatest = !toLatest || (toLatest < version && version.match(/^[0-9\.]+$/)) ? version : toLatest;
+    }
+
+    if (!toLatest) {
+      messages.push({ level: 'error', message: `Unknown conversion from ${from} to ${to}` });
+      return '';
+    }
+
+    toVersion = toLatest;
+  }
+
+  if (
+    !fs.existsSync(
+      path.resolve(__dirname, `../../templates/conversion/${from}@${fromVersion}->${to}@${toVersion}.json`)
+    )
+  ) {
+    messages.push({ level: 'error', message: `Unknown conversion from ${from}@${fromVersion} to ${to}@${toVersion}` });
+    return '';
+  }
+
+  return path.resolve(__dirname, `../../templates/conversion/${from}@${fromVersion}->${to}@${toVersion}.json`);
```

Comment:
This path is long and complicated, it would be better to extract it into one common constant.